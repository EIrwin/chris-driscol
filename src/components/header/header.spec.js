// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Component from './header.component';

it('renders correctly', () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
