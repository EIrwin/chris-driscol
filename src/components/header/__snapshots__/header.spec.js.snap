// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <div
    className="container"
  >
    <div
      className="intro-text"
    >
      <div
        className="intro-lead-in"
      >
        Welcome To My Website!
      </div>
      <div
        className="intro-heading"
      >
        It's Nice To Meet You
      </div>
      <a
        className="page-scroll btn btn-xl"
        href="#aboutme"
      >
        Learn about me
      </a>
    </div>
  </div>
</header>
`;
